packages/tasit-sdk/package.json

```diff
@@ -46,6 +46,7 @@
     "@babel/register": "^7.0.0",
     "chai": "^4.2.0",
     "ethers": "^4.0.27",
-    "mocha": "^6.0.1"
+    "mocha": "^6.0.1",
+    "node-fetch": "^2.3.0"
```

Comment:
Hmm, I think we only need it as a dep in one place, and then we can use it via `tasit-action` here?

Reply:
Note: We are using `node-fetch` as devDependency on `tasit-sdk` to test if an estates' image is a blank one.
Said that, I think that we really can add the dependency to one of the packages and it'll work for the others because of the `lerna --hoist` option:

> Lerna will hoist dependencies which are used in multiple projects, even if they are not used in all projects. [(refs)](https://github.com/lerna/lerna/blob/master/doc/hoist.md#forgetting-dependencies)

It's a trick that will work for `devDependencies` but it's putted as a `Disadvantages with hoisting` because it's a error prone if used with non-dev dependencies.

I'm not sure if I undestood well what you mean.

Reply:
I was saying it wouldn't need to work for the others if we stuck to always using it indirectly via some function we expose from the original package it's used in

Reply:
If we did want to use it directly in multiple spots, I think we could only count on hoisting without making it an explicit dependency everywhere we wanted it if it were used in at least 2 of n packages, not 1 of n

Reply:
But I think exposing it via a helper function to use in the Tasit SDK tests or scripts might be too much work to DRY it up for now

Reply:
Done.

Reply:
Wait, so after the removal of the dep from the Tasit SDK package, aren't we in a 1-of-n situation now? So we'd need to change something else too, right?

Reply:
Hmmm since `lerna bootstrap` is made with the `--hoist` option, all deps are installed on `TasitSDK/node_modules` and accessible for all packages.

Reply:
I don't think all deps are installed on TasitSDK node_modules. Just those with at least 2 packages using them.

https://github.com/lerna/lerna/blob/master/doc/hoist.md

If the Tasit SDK is going to directly use the `node-fetch`, I'd rathre add it back as a dev dependency

Reply:
@marcelomorgado I think this is the only thing gating merging

Reply:
Weird, here even only `tasit-action` having it as a dependency the module is inside of `TasitSDK/node_modules` and the `tasit-sdk` script is working.
Should I back the dependency to `tasit-sdk` anyway?

Reply:
I wonder if that's due to the last time we ran lerna hoist with a different package.json?

Reply:
Yeah, I think it's worth adding the dependency back if we'll use it directly for development in that package

Reply:
> I wonder if that's due to the last time we ran lerna hoist with a different package.json?

I didn't get it


> Yeah, I think it's worth adding the dependency back if we'll use it directly for development in that package

Done.

Reply:
> I didn't get it

If we ran the `lerna --hoist` command most recently before editing the `package.json` file and rerunning the command, then it would make sense that `node-fetch` would still have been hoisted to the root of Tasit SDK.

Reply:
Hmmm, I've run the clean+bootstrap script after removal of `node-fetch` from `tasit-sdk`

Reply:
Ah ok, then that's probably not the explanation. Does the clean command delete the 1st-party `node_modules` folders from our packages?

Reply:
Yes.

Reply:
Ah ok, good to know. Feel free to resolve this thread.

Reply:
Resolving this.